tools/osv-linter/internal/checks/affected.go

```diff
@@ -0,0 +1,21 @@
+package checks
+
+import (
+	"github.com/tidwall/gjson"
+)
+
+var CheckAffectedFieldValid = &CheckDef{
+	Code:        "A0001",
```

Comment:
(Over) thinking about codes, and prefixes, IMO, this is a record-level check, but I'm already using "R" for ranges...

Reply:
I also feel I shouldn't name this file as `affected.go`?. `record.go` sounds better and can add more checks in the future.

Reply:
I renamed the file to `record.go` and renamed the check to `CheckRecordHasAffected`, but i didn't change the code. I agree that this is a record-level check and "R" is better than "A". Should we rename the range code to "V" for versions?

Reply:
> Should we rename the range code to "V" for versions?

Versions are one of the types of things in ranges :-)

Maybe for record-level, we go with the very non-obvious "O" for OSV? Naming is hard...